tools/ci_build/github/pai/migraphx_test_launcher.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+build_dir=${1:-"."}
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+echo "Warning: The following tests are EXCLUDED on MIGraphX agent:"
+gtest_filter="-"
+while read line; do
+  gtest_filter="$gtest_filter:$line"
+  echo "$line"
+done <$script_dir/migraphx-excluded-tests.txt
+echo ""
+
+echo "Running ./onnxruntime_test_all .."
+$build_dir/onnxruntime_test_all --gtest_filter=$gtest_filter
```

Comment:
>onnxruntime_test_all  [](http://example.com/codeflow?start=11&length=21)

Does it run InferenceSessionTests.* under 
./onnxruntime/test/framework/inference_session_test.cc
? If not, perhaps in future we can consider to run those instead. If so, perhaps we can include ifdef USE_MIGRAPHX in the beginning. Right now, there's USE_ROCM there.

Reply:
It's run part of InferenceSessionTests.* tests, but not included the GPU part. We can add them if we need.